flexget/plugins/api_trakt.py

```diff
@@ -294,6 +294,37 @@ def get_id(type, name=None, year=None, trakt_slug=None, tmdb_id=None, imdb_id=No
 
     # TODO: Fall back to name/year search
 
+    clean_title = normalize_series_name(name)
+    found = None
+    try:
+        results = req_session.get(get_api_url('search'), params={'query': clean_title, 'type': type}).json()
+    except requests.RequestException as e:
+        log.debug('ID Lookup failed on error: %s' % e)
+    if results['type'] is type:
+        for result in results:
+            if clean_title == result[type]['title']:
```

Comment:
for all I care comment it out for now. I know I've done a few searches for some titles of stuff that I watch. and each one come up within the first grouping of searches. And I forgot to normalize the json title